### PR TITLE
Update 2024 Template (MyST)

### PR DESCRIPTION
**Pull request by rowanc1**
Initial template changes for MyST Markdown, please wait on merging in as this will be an initial test of the new publishing infrastructure!

*Editor*: 

*Reviewers*:

### CODE REVIEW

**Review by cbcunc — APPROVED** [review]
Thank you, @rowanc1

File: papers/00_bibderwalt/main.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+Matplotlib [@matplotlib],
+NumPy [@numpy],
+pandas [@pandas1; @pandas2],
+scikit-learn [@sklearn1; @sklearn2], and
+SciPy [@scipy]
+are included in this paper's `.bib` file.
+
+In this paper we not only terraform a desert using the package terradesert [@terradesert], we also catch a sandworm with it.
+To cite a website, the following BibTeX format plus any additional tags necessary for specifying the referenced content is recommended.
+If you are citing a team, ensure that the author name is wrapped in additional braces `{Team Name}`, so it is not treated as an authors first and last names.
```
Comment (mepa) [inline papers/00_bibderwalt/main.md]:
```suggestion
If you are citing a team, ensure that the author name is wrapped in additional braces `{Team Name}`, so it is not treated as an author's first and last names.
```

File: papers/00_bibderwalt/mybib.bib [hunk 2]
```diff
@@ -1,92 +1,91 @@
...
-  address =     "Indianapolis, IN",
-  publisher =   "Hackett",
-  doi = "10.1017/CBO9780511808432",
+@book{hume48,
+  author    = "David Hume",
+  year      = "1748",
+  title     = "An enquiry concerning human understanding",
+  address   = "Indianapolis, IN",
+  publisher = "Hackett",
+	doi       = {https://doi.org/10.1017/CBO9780511808432},
```
Comment (mepa) [inline papers/00_bibderwalt/mybib.bib]: Is the extra indent before `doi` intentional?

Comment (rowanc1) [reply papers/00_bibderwalt/mybib.bib]: Looks like an unintentional tab! Fixed locally.

**Pull request merged**